packages/web-shared/components/ContentSingle.js

```diff
@@ -11,14 +11,77 @@ import FeatureFeedComponentMap from './FeatureFeed/FeatureFeedComponentMap';
 import { add as addBreadcrumb, useBreadcrumbDispatch } from '../providers/BreadcrumbProvider';
 import { set as setModal, useModal } from '../providers/ModalProvider';
 
-import { Box, Loader, Longform, H3, ContentCard, BodyText, ShareButton } from '../ui-kit';
+import {
+  Box,
+  Loader,
+  Longform,
+  H3,
+  ContentCard,
+  BodyText,
+  ShareButton,
+  Icons,
+  PhospherIcon,
+} from '../ui-kit';
 import { useFeatureFeed, useHTMLContent, useVideoMediaProgress } from '../hooks';
 import { Title, ParentTitle, ParentSummary } from './ContentSingle.styles';
 
 import VideoPlayer from './VideoPlayer';
 
 import InteractWhenLoaded from './InteractWhenLoaded';
 import TrackEventWhenLoaded from './TrackEventWhenLoaded';
+import styled from 'styled-components';
+
+const infoDivider = (
+  <BodyText color="text.tertiary" mx="xs" display={{ xs: 'none', sm: 'block' }}>
+    |
+  </BodyText>
+);
+
+const CalendarIconWrapper = styled.span`
+  margin-right: ${({ theme }) => theme.space.xxs};
+  display: inline-block;
```

Comment:
![Screenshot 2024-02-22 at 12 17 06 PM](https://github.com/ApollosProject/apollos-embeds/assets/72768221/2a67b172-fe7d-4e17-ac6e-68152c43f2b4)

I think make this `inline-flex`. I'll drop another couple of changes.